### ISSUE TITLE
On incognito window there is no way to monitor active downloads

### ISSUE THREAD

**Issue by 44iCdnz** [snaildos/Fifo-Browser#162]
When you clicked a download link, it silently downloads the file without notifying you. If you go to Downloads on the menu you will find it's greyed out. The only thing you could do is go to the Downloads directory and checking the file growing in size. You can only guess if the download was completed when the file size stopped growing and equals to the file size the download page tells you (if it tells you about that at all).

**Comment by snaildos** [comment]
👍 Confirmed. I'll be looking into this.

**Comment by 3oheicrw** [comment]
> +1 Confirmed. I'll be looking into this.

On normal window, there is no way to clear the downloaded list. The list is just there and you can't do anything with it.

**Comment by snaildos** [comment]
> > +1 Confirmed. I'll be looking into this.
> 
> On normal window, there is no way to clear the downloaded list. The list is just there and you can't do anything with it.

Yeah, I probably wont add an option to clear it. Remember, I need to focus on fixing issues rather then creating new features that will cause more issues :D
